src/lib/components/shared/hover_expand.svelte

```diff
@@ -0,0 +1,15 @@
+<script lang="ts">
+    import ScrollArea from "./scroll_area.svelte";
+
+    let klass = "";
+    export { klass as class };
+    export let height = "";
+
+    $: base_class = `${klass} ${height}`;
```

Comment:
You arent declaring `base_class`

Reply:
Yes, bad?
well I saw this in skele:
https://github.com/skeletonlabs/skeleton/blob/72bd820dc65fefa61523a7d71eef6feed354dfed/packages/skeleton/src/lib/components/Avatar/Avatar.svelte#L43